### PR TITLE
feat: early access banner in header

### PR DESCRIPTION
**Pull request by jhult**

### CODE REVIEW

File: src/common/search.rs [hunk 1]
```diff
@@ -110,7 +110,7 @@ pub fn GlobalSearchBar() -> impl IntoView {
 
     view! {
         <PreSectionContainer>
-            <div class="mx-2 my-2 md:mx-0 md:w-full -mt-2 relative align-stretch flex items-center">
+            <div class="mx-2 my-2 md:mx-0 md:w-full mt-5 relative align-stretch flex items-center">
```
Comment (n1tranquilla) [inline src/common/search.rs:113]: This change doesn't seem related

**Review by n1tranquilla — APPROVED** [review]

File: src/common/search.rs (hunk 1 above)
Comment (jhult) [reply src/common/search.rs:113]: The search bar alignment needed to be adjusted to be inside the gray background (since it was no longer against the top.

Comment (n1tranquilla) [reply src/common/search.rs:113]: got it
